adapters/amqp-vertx/src/main/java/org/eclipse/hono/adapter/amqp/AmqpAdapterSaslAuthenticatorFactory.java

```diff
@@ -294,7 +294,7 @@ private void verifyExternal(final Handler<AsyncResult<DeviceUser>> completer) {
                 tenantTracker
                         .compose(tenant -> {
                             try {
-                                final TrustAnchor trustAnchor = tenantTracker.result().getTrustAnchor();
+                                final TrustAnchor trustAnchor = tenantTracker.result().getTrustAnchors().get(0);
```

Comment:
I thought the idea is to use (potentially) multiple trust anchors for validating the client certificate, isn't it?

Reply:
the idea was to leave out this part for another PR since this one is already too big to review. do you want me to add that in this PR?

Reply:
yes, exactly